qutebrowser/browser/commands.py

```diff
@@ -1639,6 +1639,7 @@ def on_editing_finished(self, elem, text):
         """
         try:
             elem.set_value(text)
+            mainwindow.raise_window(objreg.last_focused_window())
         except webelem.OrphanedError as e:
             message.error('Edited element vanished')
         except webelem.Error as e:
```

Comment:
Just a note to myself: After merging this, I should edit the line after this one to use `message.error`, as raising a `CommandError` from here seems wrong!